CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
 ## 3.72.0 (Unreleased)
+IMPROVEMENTS:
+* container: Allowed specifying a cluster id field for `google_container_node_pool.cluster` to ensure that a node pool is recreated if the associated cluster is recreated. ([#3314](https://github.com/hashicorp/terraform-provider-google-beta/pull/3314))
+* storage-transfer: added support for `azure_blob_storage_data_source` to `google_storage_transfer_job` ([#3316](https://github.com/hashicorp/terraform-provider-google-beta/pull/3316))
```

Comment:
should this be `storage` maybe?

Reply:
we've used a mix of `storage` and `storagetransfer` in the past. Considering this is its [own service](https://cloud.google.com/storage-transfer/docs/reference/rest) we should probably use `storagetransfer`